inline-r/tests/test-qq.hs

```diff
@@ -116,4 +116,13 @@ main = H.withEmbeddedR H.defaultConfig $ H.runRegion $ do
     let utf8string = "abcd çéõßø"
     io . assertEqual "" utf8string =<< fromSEXP <$> R.cast (sing :: R.SSEXPTYPE 'R.String) <$> [r| utf8string_hs |]
 
+
+    -- Disable gctorture, otherwise test takes too long to execute.
```

Comment:
Any reason why? The computation in this test (and the number of allocations) looks tiny.